src/encoding/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './slotKey';
```

Comment:
We should use same casing for files
```suggestion
export * from './slot-key';
```
or 
```suggestion
export * from './slotkey';
```

Reply:
oops missed that! fixed.